app/assets/stylesheets/app/quick_reservation.scss

```diff
@@ -0,0 +1,108 @@
+body.quick_reservation {
+  div#content {
+    h1 {
+      font-size: 2em;
+      border: none;
+    }
+    h1.gray {
+      color: #595959;
+      font-size: 1.5em;
+      margin-bottom: -1em;
+    }
+    h2 {
+      font-size: 1.5em;
+    }
+    div.reservation_notice {
+      h1 {
+        margin-bottom: 0;
+      }
+      h2 {
+        margin: 0;
+        line-height: 1em;
+      }
+      width: 100%;
+      display: flex;
+      flex-direction: column;
+      align-items: stretch;
+      justify-content: center;
+      text-align: center;
+      p {
+        padding: 0 15%;
+        color: #595959;
+        font-weight: bold;
+      }
+    }
+    dl {
+      width: 100%;
+      dt {
+        color: #595959;
+        clear: both;
+      }
+      dd {
+        margin: 0;
+        font-weight: bold;
+        margin-bottom: 1em;
+        clear: both;
+      }
+      dd.underline {
+        border-bottom: 3px solid #005B8A;
+        float: left;
+      }
+    }
+    a.end_link {
+      background-color: #005B8A;
```

Comment:
This was `$linkColor` but I changed it to get the accessibility spec to pass. Not sure if we want to update that variable value, or create an accessibility compliant alternative variable